src/components/molecules/MenuList/MenuList.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { MenuList } from "./MenuList";
+
+export default {
+  component: MenuList,
+  parameters: {
+    controls: { expanded: true },
+  },
+  argTypes: {
+    navigationType: {
+      type: "string",
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div className="w-fit">
+        <Story />
+      </div>
+    ),
+  ],
+} satisfies Meta<typeof MenuList>;
+
+export const Default: StoryObj<typeof MenuList> = {
+  argTypes: {
+    navigationType: {
+      type: "string",
+      options: ["beyond", "connection", "account"],
+      control: {
+        type: "select",
+      },
+    },
+  },
+  args: {
+    navigationType: "",
+  },
```

Comment:
Since the default story doesn't render anything, how about removing it and adding the select type to the overall `argTypes`?
```suggestion
  argTypes: {
    navigationType: {
      type: "string",
      options: ["beyond", "connection", "account"],
      control: {
        type: "select",
      },
    },
  },
  decorators: [
    (Story) => (
      <div className="w-fit">
        <Story />
      </div>
    ),
  ],
} satisfies Meta<typeof MenuList>;
```